CMake/FindThrift.cmake

```diff
@@ -1,3 +1,16 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 # Copyright 2012 Cloudera Inc.
```

Comment:
I meet the license header failed exception, but seems like the header is really not same with the check, how do I do to fix it? The header required `Copyright (c) Facebook`, but it is `Copyright 2012 Cloudera Inc.`. @pedroerp